modules/drivers/oracle/src/metabase/driver/oracle.clj

```diff
@@ -627,7 +627,8 @@
 
 (defmethod unprepare/unprepare-value [:oracle ZonedDateTime]
   [_ t]
-  (format "timestamp '%s'" (t/format "yyyy-MM-dd HH:mm:ss.SSS VV" t)))
+  (format "timestamp '%s'" (-> (t/format "yyyy-MM-dd HH:mm:ss.SSS VV" t)
+                               (str/replace #"Z$" "UTC"))))
```

Comment:
This is inspired by `unprepare/unprepare-value [:oracle OffsetDateTime]`, one method above in code.

Reply:
Thank you.